bin/plugin/commands/test/fixtures/pull-requests.json

```diff
@@ -3097,7 +3097,7 @@
 			"diff_url": "https://github.com/WordPress/gutenberg/pull/33901.diff",
 			"patch_url": "https://github.com/WordPress/gutenberg/pull/33901.patch"
 		},
-		"body": "<!-- Learn the overall process and best practices for pull requests at https://github.com/WordPress/gutenberg/blob/HEAD/docs/contributors/repository-management.md#pull-requests. -->\r\n\r\n<!-- Gutenberg's license is in the process of updating to be dual-licensed under the GPL and MPL. As part of that transition, all new contributions are dual-licensed. For more information, see: https://github.com/WordPress/gutenberg/blob/trunk/LICENSE.md -->\r\n\r\n## Description\r\n<!-- Please describe what you have changed or added -->\r\n\r\nIt's the result of running:\r\n\r\n```bash\r\nnpm run lint-js -- --fix\r\n```\r\n\r\nIt was applied to files that had reported violations for `jsdoc/check-line-alignment` rule.",
+		"body": "<!-- Learn the overall process and best practices for pull requests at https://github.com/WordPress/gutenberg/blob/HEAD/docs/contributors/repository-management.md#pull-requests. -->\r\n\r\n<!-- Gutenberg's license is in the process of updating to be dual-licensed under the GPL and MPL. As part of that transition, all new contributions are dual-licensed. For more information, see: https://github.com/WordPress/gutenberg/blob/trunk/LICENSE.md -->\r\n\r\n## Description\r\n<!-- Please describe what you have changed or added -->\r\n\r\nIt's the result of running:\r\n\r\n```bash\r\nnpm run lint:js -- --fix\r\n```\r\n\r\nIt was applied to files that had reported violations for `jsdoc/check-line-alignment` rule.",
```

Comment:
It's a test fixture so you can ignore this file.

Reply:
Fixed with:  ddf7f51 / f79530f 👍